.github/workflows/main.yml

```diff
@@ -103,3 +103,12 @@ jobs:
         run: |
           qmake
           nmake
+
+  # If this fails, consider changing your text or adding something to .typos.toml.
```

Comment:
Not sure I get this comment, does this mean if the CI reports a spelling mistake, you should add an exception to `.typos.toml`?

Reply:
Yes, it is boilerplate from all of our other repo CIs.